updater/spec/dependabot/updater/operations/refresh_security_update_pull_request_spec.rb

```diff
@@ -113,6 +113,7 @@
 
   before do
     allow(Dependabot::Experiments).to receive(:enabled?).with(:bundler_v1_unsupported_error).and_return(false)
+    allow(Dependabot::Experiments).to receive(:enabled?).with(:lead_security_dependency).and_return(false)
```

Comment:
I would suggest turning the experiment on in these tests as we believe this is the right way to do security updates with a single dependency going forward.

Reply:
added a new test for single dependency update, this particular test case is intended towards deprecation notices